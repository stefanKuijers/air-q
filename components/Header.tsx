import Link from 'next/link';
import React from 'react';

const linkStyle = {
    marginRight: 15,
};

const Header: React.FunctionComponent = () => (
    <div>
        <Link href="/">
            <a style={linkStyle}>Home</a>
        </Link>
        <Link href="/about">
            <a style={linkStyle}>About</a>
        </Link>
        <Link href="/blog">
            <a style={linkStyle}>Blog</a>
        </Link>
        <Link href="/shows">
            <a style={linkStyle}>Shows</a>
        </Link>
        <Link href="/inspiration">
            <a style={linkStyle}>Inspiration</a>
        </Link>
    </div>
);

export default Header;
