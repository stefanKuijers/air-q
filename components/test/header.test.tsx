import { render, RenderResult } from '@testing-library/react';
import React from 'react';

import Header from '../Header';

const setup = (component: React.ReactElement): RenderResult => {
    const tree = render(component);
    // const commentButton = tree.container.querySelector('button');
    return {
        ...tree,
    };
};

describe('<Header />', () => {
    it('matches snapshot', async () => {
        const { container } = setup(<Header />);
        expect(container).toMatchSnapshot();
    });
});
