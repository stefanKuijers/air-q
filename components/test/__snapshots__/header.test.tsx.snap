// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> matches snapshot 1`] = `
<div>
  <div>
    <a
      href="/"
      style="margin-right: 15px;"
    >
      Home
    </a>
    <a
      href="/about"
      style="margin-right: 15px;"
    >
      About
    </a>
    <a
      href="/blog"
      style="margin-right: 15px;"
    >
      Blog
    </a>
    <a
      href="/shows"
      style="margin-right: 15px;"
    >
      Shows
    </a>
    <a
      href="/inspiration"
      style="margin-right: 15px;"
    >
      Inspiration
    </a>
  </div>
</div>
`;
